plugins/ipam/config/config.go

```diff
@@ -107,6 +108,10 @@ func LoadIPAMConfig(bytes []byte, args string) (*IPAMConfig, string, error) {
 		}
 	}
 
+	if ipamConf.IPAM.ID == "" {
```

Comment:
In the larger scheme of things, should we make this a required field? If an ID is not provided, the worst thing that can happen is that multiple ip addresses will map to same value (blank string). Sounds like we are imposing an additional and unnecessary complexity by making this a required field.